dhis-2/dhis-support/dhis-support-system/src/main/java/org/hisp/dhis/system/util/MathUtils.java

```diff
@@ -71,6 +71,8 @@
 
     private static final Pattern ZERO_PATTERN = Pattern.compile( "^0(\\.0*)?$" );
 
+    private static final Random RANDOM = new Random();
+
```

Comment:
Might be better to use a ThreadLocalRandom.
See Random.class java doc: Consider instead using java.util.concurrent.ThreadLocalRandom in multithreaded designs.

Reply:
Thanks, I'll check it out!